R/logger.R

```diff
@@ -275,6 +275,20 @@ log_namespaces <- function() {
 }
 
 
+#' Returns number of currently active indices
+#' 
+#' @param namespace string referring to the `logger` environment /
+#'   config to be used to override the target of the message record to
+#'   be used instead of the default namespace, which is defined by the
+#'   R package name from which the logger was called, and falls back
+#'   to a common, global namespace.
```

Comment:
No need to copy the whole `namespace` param description, as you can reference it from another fn to be reused, e.g.:

```suggestion
#' @inheritParams get_logger_definitions
```